<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Device name: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="chkBoxEnableTray">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Enable tray icon</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="chkBoxCheckUpdates">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Check for updates</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update interval (h):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="spnBoxUpdateInterval">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>12</number>
         </property>
         <property name="maximum">
          <number>2400</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lnEdtdeviceName"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Transfer options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lnEdtDirectory"/>
         </item>
         <item>
          <widget class="QPushButton" name="pshBtnSelectDir">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spnBoxTransferPort">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65536</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Directory:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Buffer:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spnBoxBuffer">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65536</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblTransferTimeout">
         <property name="text">
          <string>Timeout (s):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="spnBoxTransferTimeout">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65536</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Broadcast</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spnBoxBroadcastPort">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65536</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Timeout (s):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spnBoxBroadcastTimeout">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65536</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Interval (s):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spnBoxBroadcastInterval">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65536</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkBoxAdvanced">
     <property name="text">
      <string>Advanced</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBoxAdvanced</sender>
   <signal>clicked(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>onChkBoxAdvancedClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBoxAdvanced</sender>
   <signal>clicked(bool)</signal>
   <receiver>lblTransferTimeout</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBoxAdvanced</sender>
   <signal>clicked(bool)</signal>
   <receiver>spnBoxTransferTimeout</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBoxCheckUpdates</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_8</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBoxCheckUpdates</sender>
   <signal>clicked(bool)</signal>
   <receiver>spnBoxUpdateInterval</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onChkBoxAdvancedClicked(bool)</slot>
 </slots>
</ui>
